tests/ha/rear_restore.pm

```diff
@@ -23,7 +24,18 @@ sub run {
     assert_screen('rear-boot-screen');
     send_key_until_needlematch('rear-recover-selected', 'up');
     send_key 'ret';
-    $self->wait_boot_past_bootloader;
+    # Handle the output error for tty0 in ppc64, more details here TEAM-9655
```

Comment:
Not everybody has access to our ticketing system. Best to not reference it in the code. More details about why a simple `wait_boot_past_bootloader` is not enough on ppc64le would be nice to add here as well.